libraries/elkstack.rb

```diff
@@ -0,0 +1,40 @@
+# Module for reusable code for elkstack
+module Elkstack
```

Comment:
It's based on https://github.com/rackspace-cookbooks/platformstack/blob/master/libraries/platformstack.rb, the goal is to support old platformstack attributes and new elkstack attributes. So upgrading to this ELKstack version would not change (almost) the current behavior.

By example I could have `node['platformstack']['cloud_monitoring']['enabled']` to enable/disable cloudmonitoring because I have platformstack in my dependencies, or I could define `node['elkstack']['config']['cloud_monitoring']['enabled']` to enable/disable cloudmonitoring